css/style.css

```diff
@@ -23,7 +23,7 @@ h2{font-family: 'Franklin Gothic Medium';
 div{
 display:block;
 }
-  .input-radio,
+  .input-radio,,
```

Comment:
Kindly remove this class from your code base as it is empty and doesn't contain any properties for that class to style it.
Thank you🙂